script/component-migration/transformers/progressbar.js

```diff
@@ -0,0 +1,5 @@
+function progressBarReplacement() {
+  return ['va-progress-bar', {}];
+}
```

Comment:
Can you add a comment saying that the object is empty because the props are staying the same?